src/main/java/org/truffleruby/interop/OutgoingForeignCallNode.java

```diff
@@ -113,6 +123,14 @@ protected OutgoingNode createHelperNode(int argsLength) {
             return new CallOutgoingNode(argsLength);
         } else if (name.equals("new")) {
             return new NewOutgoingNode(argsLength);
+        } else if (name.equals("to_a") || name.equals("to_ary")) {
+            return new ToAOutgoingNode();
+        } else if (name.equals("respond_to?")) {
+            return new RespondToOutgoingNode();
+        } else if (name.equals("inspect")) {
+            return new InspectOutgoingNode();
+        } else if (name.equals("send")) {
```

Comment:
Should we handle the `__send__` alias too? Or maybe only handle `__send__` as `send` could be a rather common method name.